### PR TITLE
Fix: vtx tab -> vtx type

### PR DESCRIPTION
**Pull request by limonspb**
At some point, smartaudio version stopped being detected by the configurator 10.9. This is a fix.
Should be cherry-picked on 10.9-maintenance if approved.
image

@chmelevskij @haslinghuis  @blckmn

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
